### DISCUSSION TITLE
Any chance the Synology drivers work on DS1520+ as well?

### DISCUSSION THREAD

**Discussion by WMuellner** [bb-qq/aqc111#134] [Ideas]
I've got a Synology DS1520+ which use an Intel Celeron J4125 CPU (Geminilake architecture). Is there any chance that these drivers work on this architecture as well? Or are there any plans to port the code to this platform?

br, Walter

**Comment by bb-qq** [comment]
This driver should work.
https://github.com/bb-qq/r8152/releases/download/2.18.1-1/r8152-geminilake-2.18.1-1_7.2.spk
